### ISSUE TITLE
How to render SMPLX front and back normal if I konw the camera parameters KRT and SMPLX parameters?

### ISSUE THREAD

**Issue by fishfishson** [YuliangXiu/ECON#102]
Dear author, 

I want to test your normal net performance with my custom multiview data. Since I have already konwn the camera parameters and SMPLX parameters of the captured human, could you pls tell me how to render the SMPLX normal so that the results are consistent with yours?

**Comment by fishfishson** [comment]
> Please check [infer.py#L201](https://github.com/YuliangXiu/ECON/blob/master/apps/infer.py#L201) to know how ECON renders the SMPL-X normal maps from SMPL-X mesh.

Should I convert perspetive camera to orthgonal camera?

**Comment by YuliangXiu** [comment]
Yes, all the 2D projections of ECON/ICON are orthogonal or weak perspective. 

Please check [TestDataset.py#L186](https://github.com/YuliangXiu/ECON/blob/master/lib/dataset/TestDataset.py#L186) to see the camera details to translate/scale the estimated 3D SMPL-X body.

**Comment by notorious-eric** [comment]
> Yes, all the 2D projections of ECON/ICON are orthogonal or weak perspective.
> 
> Please check [TestDataset.py#L186](https://github.com/YuliangXiu/ECON/blob/master/lib/dataset/TestDataset.py#L186) to see the camera details to translate/scale the estimated 3D SMPL-X body.

Hi I also use perspective camera and there is still some mismatch between the rendered normal and RGB. would you explain how can I find the scale and tran in "scale, tranX, tranY = preds_dict["cam"].split(1, dim=1)" to translate/scale the estimated 3D SMPL-X body. When i read smpl model there is no such thing. Thank you.
